sdk/template/template-dpg/assets.json

```diff
@@ -0,0 +1,6 @@
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath": "js",
+  "TagPrefix": "js/template/template-dpg",
+  "Tag": ""
```

Comment:
I think we will need to push recordings and create a tag, `dev-tool test-proxy push`.

Reply:
Pushed :) thanks